src/menu/menu.js

```diff
@@ -219,7 +219,11 @@ function renderDropdownItems(items, pm) {
   let rendered = []
   for (let i = 0; i < items.length; i++) {
     let inner = items[i].render(pm)
-    if (inner) rendered.push(elt("div", {class: prefix + "-dropdown-item"}, inner))
+    if (inner) {
+      let element = elt("div", {class: prefix + "-dropdown-item"}, inner)
+      if (items[i].command_) element.setAttribute("data-cmd", items[i].command_.name)
```

Comment:
Originally I did that, but during styling traversing within sibling nodes was much much easier